frontend/src/store/user.js

```diff
@@ -22,6 +29,23 @@ export const registrantSlice = createSlice({
   },
 });
 
+export const loginSlice = createSlice({
+  name: "login",
+  initialState: {
+    value: loginInitialState,
+  },
+  reducers: {
+    login: (state, action) => {
+      state.value = action.payload;
+    },
+  },
+});
```

Comment:
This is just a tip. 
Best to use redux is when you are handling global state management.
Like user details information handling.
So in this login/register scenario, it is not advisable because its an overkill or over engineered.
You can just use fetch/axios directly in this part.
So right now, I will let you use redux but next time, try to use the simpler one.